app-misc/jigit/jigit-1.20.recipe

```diff
@@ -0,0 +1,59 @@
+SUMMARY="A tool to ease the distribution of very large files"
+DESCRIPTION="Jigsaw Download, or jigdo, is a tool designed to ease the \
+distribution of very large files over the internet such as CD or DVD \
+images. Its aim is to make downloading these images as easy as a click \
+on a direct download link in a browser, while avoiding all the problems \
+that server administrators could have with hosting large files."
+HOMEPAGE="http://www.einval.com/~steve/software/JTE/"
+COPYRIGHT="2014 Steve McIntyre"
```

Comment:
Maybe:
```diff
-COPYRIGHT="2014 Steve McIntyre"
+COPYRIGHT="2004-2014 Steve McIntyre"
```
(I'm not sure about this. In particular, I don't know if the project was initiated by other authors in 2004.)

Reply:
"George Danchev" and "Thomas Schmitt" probably need to be included here too (for libjte)